.github/workflows/build_win.yml

```diff
@@ -0,0 +1,78 @@
+name: build_windows
+
+on:
+  push:
+    tags:
+      - "v*"
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2.5.0
+      - name: Setup node.js
+        uses: actions/setup-node@v3.5.1
+      - name: Setup Protobuf
+        run: choco install protoc
+      - name: Setup rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Download Pre Built Media Libraries Zip
+        uses: robinraju/release-downloader@v1.6
+        with:
+          repository: "MirrorX-Desktop/media_libraries_auto_build"
+          tag: "latest"
+          fileName: "artifacts_windows_x64.zip"
+      - name: Unzip Pre Built Media Libraries
+        run: |
+          7z x artifacts_windows_x64.zip -oartifacts_windows_x64
+      - name: ls
+        run: |
+          Get-ChildItem
+      - name: Set Pre Built Media Libraries Path
+        run: |
+          Add-Content -Path $env:GITHUB_ENV -Value "MIRRORX_MEDIA_LIBS_PATH=${{github.workspace}}\\artifacts_windows_x64"
+      - name: Setup tauri-cli
+        uses: baptiste0928/cargo-install@v1
+        with:
+          crate: tauri-cli
+          version: "1.2.0"
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v3
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: build
+        run: |
+          cd mirrorx
+          yarn install
+          cd ..
+          cargo tauri build
+      - name: Zip
+        run: |
+          7z a MirrorX_x64.zip ${{github.workspace}}\\target\\release\\MirrorX.exe
+      - name: Release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "MirrorX_x64.zip"
+          tag: ${{github.ref_name}}
+          allowUpdates: true
```

Comment:
What's purpose of this file? Does it has any differences with build_windows.yml?